libr/include/r_util.h

```diff
@@ -41,7 +41,7 @@ int gettimeofday (struct timeval* p, void* tz);
 #include "r_util/r_des.h"
 #include "r_util/r_file.h"
 #include "r_util/r_hex.h"
-#include "r_util/r_log.h"
+#include "r_util/r_logging.h"
```

Comment:
i would keep r_log.h as well as the api names

Reply:
Should we rename core/log.c to something more descriptive then to avoid conflicts? There's also a lot of existing usage of `r_core_log` elsewhere related to it. Don't want to make it too confusing having a `r_core_log` and `r_log`.